doc/nrf/config_and_build/configuring_app/kconfig/index.rst

```diff
@@ -54,16 +54,29 @@ This means that this file is available when building the application until you c
       See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run menuconfig or guiconfig.
       To locate a specific configuration option, use the **Jump to** field.
 
+.. _configuration_temporary_change_single_build:
+
+Temporary Kconfig changes for a single build
+============================================
+
+You can also apply temporary Kconfig changes to a single build by :ref:`providing the value of a chosen Kconfig option as a CMake option <cmake_options>`.
+The Kconfig option configuration will be available until you clean the build directory pristinely.
```

Comment:
```suggestion
The Kconfig option configuration will be available until you clean the build directory.
```
The cleaning itself doesn't have to be pristine.

Reply:
Yes, but the option is being applied until you clean pristinely.

Usage of this word here was very adamantly defended by Torsten Rasmussen when I was working on this piece of docs around 2 years ago, so I will keep this pristinely.